ruby/vampires.rb

```diff
@@ -11,49 +11,64 @@ def detection
 	
 	puts "How old are you? What year were you born? "
 	 age = gets.chomp
-   if age < "100 "
+   if age.to_i < 100
```

Comment:
Check out the Ruby Time class and see what you can do with `Time.now.year`.  This will help the code run a but more dynamically.  Otherwise you will have to keep changing this file every new year to keep the code running properly.